pkg/trait/cron.go

```diff
@@ -475,3 +479,31 @@ func checkedStringToUint64(str string) uint64 {
 	}
 	return res
 }
+
+// changeSourcesCronURI is in charge to change the value of the from route with a component that executes
+// the workload just once.
+func (t *cronTrait) changeSourcesCronURI(e *Environment) error {
```

Comment:
so this works my manipulating the source of the given integration which I would like to avoid as it is quite fragile and I rather do it in the underlying runtime but it requires some[ changes on camel-core](https://issues.apache.org/jira/browse/CAMEL-20409) and camel-k quarkus extension/spring boot starter

Reply:
Depends on https://github.com/apache/camel-quarkus/issues/5745